system-test/bigtable.js

```diff
@@ -388,18 +388,19 @@ describe('Bigtable', function() {
         assert.ifError(err);
         assert.strictEqual(families.length, 3);
         assert(families[0] instanceof Family);
-        assert.strictEqual(families[0].name, FAMILY.name);
+        assert.strictEqual(families[2].id, FAMILY.id);
```

Comment:
Why was there a change from `families[0]` to `families[2]`?

Reply:
because 2 families("follow", "traits") are already created above(ln. no. 61) in before section. And 3rd family("presidents") is created here, so index should be 2.